2006Redone Server/src/redone/discord/JavaCord.java

```diff
@@ -0,0 +1,61 @@
+package redone.discord;
+
+import org.javacord.api.DiscordApi;
+import org.javacord.api.DiscordApiBuilder;
+import org.javacord.api.entity.channel.TextChannel;
+import org.javacord.api.entity.message.MessageBuilder;
+import org.javacord.api.util.logging.ExceptionLogger;
+import redone.game.players.PlayerHandler;
+
+/**
+ * @author Patrity || https://www.rune-server.ee/members/patrity/
+ */
+
+public class JavaCord {
+
+    private static String serverName = "2006-ReBotted";
+
+    private static DiscordApi api = null;
```

Comment:
Can you instead make this quickly just read in from a file so we can add the file to .gitignore?

Reply:
done, its now loaded from data/discord.json which will be created on first run if it doesn't exist already